docs/api.rst

```diff
@@ -788,6 +788,61 @@ Query the result of an ongoing backend task
    :statuscode 409: No user is currently logged in
    :statuscode 500: Internal Rotki error
 
+Query the current price of assets
+===================================
+
+.. http:get:: /api/(version)/assets/prices/current
+
+   Querying this endpoint with a list of assets and a target asset will return a dictionary with the the price of the assets in the target asset currency. Providing an empty list or no target asset is an error.
+
+   .. note::
+      This endpoint can also be queried asynchronously by using ``"async_query": true``.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /api/1/assets/prices/current HTTP/1.1
+      Host: localhost:5042
+
+      {
+          "assets": ["BTC", "ETH", "LINK", "USD", "EUR"],
+          "target_asset": "USD"
+      }
+
+   :reqjson list assets: A list of assets to query their current price
+   :reqjson string target_asset: The asset of the price currency
```

Comment:
nitpick: asset and currency are the same thing so this should be: The target asset against which to return the price of each asset in the the list